### ISSUE TITLE
Nit problem with SASS version of "unstack" in flex grid

### ISSUE THREAD

**Issue by RioBrewster** [zurb/foundation-sites#7973]
The vanilla CSS specifies:

    .row.medium-unstack .column, .row.medium-unstack .columns {
    -webkit-flex: 0 0 100%;
      -ms-flex: 0 0 100%;
          flex: 0 0 100%; }

The compiled SASS only specifies:

  .row.medium-unstack .column {
    -webkit-flex: 0 0 100%;
      -ms-flex: 0 0 100%;
          flex: 0 0 100%; }

So class="columns" works with unstack only if you are using the css version, not SASS.

**Comment by zurbchris** [comment]
That could be UnCss stripping out `.columns` if you're not using it in your markup somewhere.

**Comment by rafibomb** [comment]
@RioBrewster Are you using the production flag? How are watching for Sass changes? It would help us get to the bottom of this.

**Comment by RioBrewster** [comment]
I use class="columns" in multiple places - Typically I use "columns" and not "column" everywhere for consistency.
That's the bug - it is NOT working with "columns" but it does work with "column".
No i'm not using the production flag.
I created the project using YETI - Basic Template.  But I had to edit the gulpfile so I'm running it with npm start.

```javascript
var gulp = require('gulp');
var $    = require('gulp-load-plugins')();

var sassPaths = [
  'bower_components/foundation-sites/scss',
  'bower_components/motion-ui/src',
  'scss'
 
];

gulp.task('sass', function() {
  return gulp.src('scss/**/**/**/*.scss')
    .pipe($.sass({
      includePaths: sassPaths
    })
      .on('error', $.sass.logError))
    .pipe($.autoprefixer({
      browsers: ['last 2 versions', 'ie >= 9']
    }))
    .pipe(gulp.dest('css'));
});

gulp.task('default', ['sass'], function() {
  gulp.watch(['scss/**/**/**/*.scss'], ['sass']);
});

```

**Comment by RioBrewster** [comment]
I use class="columns" in multiple places - Typically I use "columns" and not "column" everywhere for consistency.
That's the bug - it is NOT working with "columns" but it does work with "column".
No i'm not using the production flag.
I created the project using YETI - Basic Template.  But I had to edit the gulpfile so I'm running it with npm start.

```javascript
var gulp = require('gulp');
var $    = require('gulp-load-plugins')();

var sassPaths = [
  'bower_components/foundation-sites/scss',
  'bower_components/motion-ui/src',
  'scss'
 
];

gulp.task('sass', function() {
  return gulp.src('scss/**/**/**/*.scss')
    .pipe($.sass({
      includePaths: sassPaths
    })
      .on('error', $.sass.logError))
    .pipe($.autoprefixer({
      browsers: ['last 2 versions', 'ie >= 9']
    }))
    .pipe(gulp.dest('css'));
});

gulp.task('default', ['sass'], function() {
  gulp.watch(['scss/**/**/**/*.scss'], ['sass']);
});

```

**Comment by gakimball** [comment]
I spun up a basic project from the command line (the same one used by Yeti Launch) and when I compiled the Sass I saw the correct code. I believe 1249929b343b865c2b330e79272e2189558e432f fixed this issue.

There's currently a bug with Yeti Launch not downloading the latest version of Foundation into the project. If you have a development environment set up on your machine, open the project folder in your command line and run `bower install foundation-sites#6.1.2 --save`. That will get your project up to date.
